ducktape/command_line/main.py

```diff
@@ -176,7 +178,7 @@ def main():
         (cluster_mod_name, cluster_class_name) = args.cluster.rsplit('.', 1)
         cluster_mod = importlib.import_module(cluster_mod_name)
         cluster_class = getattr(cluster_mod, cluster_class_name)
-        session_context.cluster = cluster_class()
+        session_context.cluster = cluster_class(cluster_file = args.cluster_file)
```

Comment:
python convention - no space around the `=` in the key/val pair in kwargs of a function call